src/libraries/Microsoft.PowerFx.Core/Parser/TexlParser.cs

```diff
@@ -1766,11 +1766,12 @@ internal static string GetTokString(TokKind kind)
             }
         }
 
-        public static string Format(string text)
+        public static string Format(string text, Flags? flags = null)
         {
+            flags = flags ?? Flags.EnableExpressionChaining;

```

Comment:
Matched with the comment on the signature, just or in expression chaining, as in `flags |= Flags.EnableExpressionChaining`.  Unless there is a case, which doesn't exist today, where we should be formatting without this flag.